### DISCUSSION TITLE
WARNING: WireGuard does not seem to be running

### DISCUSSION THREAD

**Discussion by Terno2004** [angristan/wireguard-install#512] [Help]
Hello! I have this problem for install wireguard. Please help me!

root@vmi1965829:~# systemctl status wg-quick@wg0
● wg-quick@wg0.service - WireGuard via wg-quick(8) for wg0
     Loaded: loaded (/lib/systemd/system/wg-quick@.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Thu 2024-06-27 10:31:38 CEST; 5min ago
       Docs: man:wg-quick(8)
             man:wg(8)
             https://www.wireguard.com/
             https://www.wireguard.com/quickstart/
             https://git.zx2c4.com/wireguard-tools/about/src/man/wg-quick.8
             https://git.zx2c4.com/wireguard-tools/about/src/man/wg.8
   Main PID: 11550 (code=exited, status=2)

Jun 27 10:31:38 vmi1965829.contaboserver.net systemd[1]: Starting WireGuard via wg-quick(8) for wg0...
Jun 27 10:31:38 vmi1965829.contaboserver.net wg-quick[11550]: [#] ip link add wg0 type wireguard
Jun 27 10:31:38 vmi1965829.contaboserver.net wg-quick[11550]: [#] wg setconf wg0 /dev/fd/63
Jun 27 10:31:38 vmi1965829.contaboserver.net wg-quick[11550]: [#] ip -4 address add 10.66.66.1/24 dev wg0
Jun 27 10:31:38 vmi1965829.contaboserver.net wg-quick[11550]: [#] ip -6 address add fd42:42:42::1/64 dev wg0
Jun 27 10:31:38 vmi1965829.contaboserver.net wg-quick[11579]: RTNETLINK answers: Permission denied
Jun 27 10:31:38 vmi1965829.contaboserver.net wg-quick[11550]: [#] ip link delete dev wg0
Jun 27 10:31:38 vmi1965829.contaboserver.net systemd[1]: wg-quick@wg0.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Jun 27 10:31:38 vmi1965829.contaboserver.net systemd[1]: wg-quick@wg0.service: Failed with result 'exit-code'.
Jun 27 10:31:38 vmi1965829.contaboserver.net systemd[1]: Failed to start WireGuard via wg-quick(8) for wg0.

**Comment by abuwho** [comment]
I am seeing the same issue, using Google Cloud VM. Why could it be?

**Reply by angristan** [reply]
I think this might be the error you get when ipv6 is disabled, you can check using `sysctl net.ipv6.conf.all.disable_ipv6`, it should be `0`

**Reply by abuwho** [reply]
Yes, it seems to be already 0.

```
root@vpn-01:~# sysctl net.ipv6.conf.all.disable_ipv6
net.ipv6.conf.all.disable_ipv6 = 0
```
